src/main/modules/helmet/modules/contentSecurityPolicy.ts

```diff
@@ -9,7 +9,7 @@ export class ContentSecurityPolicy {
   constructor (public developmentMode: boolean) {}
 
   enableFor (app: express.Express) {
-    const scriptSrc = [self, '*.google-analytics.com']
+    const scriptSrc = ['\'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=\'', '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\'', self, '*.google-analytics.com']
```

Comment:
This change does not seem to be related to the missing icon problem, what is it for?

Reply:
we noticed it when we were looking at errors returned to the console and the network tab so seemed as good a place to fix them. Am happy to pull them out into a separate PR

Reply:
I would love to have it in separate PR.